packages/tabs/src/tab.css

```diff
@@ -54,7 +54,7 @@ governing permissions and limitations under the License.
     height: auto;
 }
 
-:host([vertical]) #itemLabel {
+:host([dir][vertical]) slot[name='icon'] + #itemLabel {
```

Comment:
I'm curious about why we added the `dir` attribute selector here? I understand that v3 elements require the `dir` attribute in order to be styled properly, but it doesnt seem like the direction influences the styling in this case

Reply:
This is extending the updated root selector in `spectrum-tab.css`, allowing us the maintain the same specificity relationship between the generated output and our additional features.
```
:host([dir='ltr']) slot[name='icon'] + #itemLabel
```

Reply:
Ah okay, makes sense!